### ISSUE TITLE
A new release for branch 3.x?

### ISSUE THREAD

**Issue by truls1502** [zaporylie/php-vipps#53]
Thanks for the great work so far. 🙌🏻  Do you think it would be possible to release a version 3 based on the [3.x](https://github.com/zaporylie/php-vipps/tree/3.x) branch?

**Comment by zaporylie** [comment]
Yes, that's the plan but I need some dedicated time before I can make this happen. Currently been busy with project work.

**Comment by truls1502** [comment]
I didn't expect a quick answer. How about a beta/alpha release? Then, we could fix it incrementally until we get an official version. I am asking because the package `guzzlehttp/guzzle` is blocked at the moment. Otherwise, are there some specific areas in which you would like to get help from us contributors? :nerd_face:
